config.go

```diff
@@ -10,8 +10,20 @@ type Config struct {
 	// HTTPAddr is the HTTP server's bind address
 	HTTPAddr string `default:":5000" split_words:"true"`
 
-	// DBConnURL is the MongoDB server's connection URL
-	DBConnURL string `default:"mongodb://knative-scout-dev:secretpassword@localhost:27017" split_words:"true"`
+	// DbHost is the MongoDB server host
+	DbHost string `default:"localhost" split_words:"true"`
+
+	// DbPort is the MongoDB server port
+	DbPort int `default:"27017" split_words:"true"`
+
+	// DbUser is the MongoDB user
+	DbUser string `default:"knative-scout-dev" split_words:"true"`
+
+	// DbPassword is the MongoDB password
+	DbPassword string `default:"secretpassword" split_words:"true"`
```

Comment:
are we storing password and username in plain text format in main code or is this default and imp credentials will be in gitignored file ?

Reply:
The credentials will be placed in environment variables, they will not be anywhere in plain text.

These variables will load values from environment variables